packages/canary-features/addon/default-features.ts

```diff
@@ -176,7 +176,6 @@ export default {
   IDENTIFIERS: true,
   REQUEST_SERVICE: null,
   CUSTOM_MODEL_CLASS: null,
-  FULL_LINKS_ON_RELATIONSHIPS: true,
   RECORD_ARRAY_MANAGER_IDENTIFIERS: true,
```

Comment:
We might want to leave this as true, in case any addon/app was checking the value cc @rwjblue